benchmarks/100.webapps/120.uploader/python/cloudflare/function.py

```
@@ -0,0 +1,56 @@
+    bs = await res.bytes()
+    
+    with open(download_path, 'wb') as f:
+        f.write(bs)
+
+def handler(event):
+
+    bucket = event.get('bucket').get('bucket')
+    output_prefix = event.get('bucket').get('output')
+    url = event.get('object').get('url')
+    name = os.path.basename(url)
+    download_path = '/tmp/{}'.format(name)
```

Ruff (0.15.6)

[error] Probable insecure usage of temporary file or directory: "/tmp/{}"

(S108)